en-US/focus-ios.xliff

```diff
@@ -391,6 +391,16 @@
         <target>Close</target>
         <note>Button label used to close a menu that displays as a popup.</note>
       </trans-unit>
+      <trans-unit id="Network.Offline.ErrorMessage" xml:space="preserve">
+        <source>The Internet connection appears to be offline.</source>
```

Comment:
Nitpick, but we're supposed to use lowercase
https://acorn.firefox.com/latest/content/word-list/g-m-NIU35GX3#section-internet-55